examples/standard-actions.amp.html

```diff
@@ -75,4 +85,43 @@
   I was initially hidden.
 </div>
 
+<div class="spacer">
+</div>
+
+<h4>Subscribe to our weekly Newsletter Form</h4>
```

Comment:
I think this is too much and makes the example cluttered, let's just do a simple `<input type="text" id="input-element">` for focus and a simple `div` for `scrollTo`, no need for unrelated stuff like `amp-form`, `CLIENT_ID`, etc...